lib/rpmts.c

```diff
@@ -615,6 +616,20 @@ rpmRC rpmtsImportPubkey(const rpmts ts, const unsigned char * pkt, size_t pktlen
     if (txn == NULL)
 	return rc;
 
+    krc = pgpPubKeyLint(pkt, pktlen, &lints);
+    if (lints) {
+        if (krc != RPMRC_OK) {
+            rpmlog(RPMLOG_ERR, _("Linting certificate: %s\n"), lints);
+        } else {
+            rpmlog(RPMLOG_WARNING, _("Linting certificate: %s\n"), lints);
```

Comment:
I'd just leave the "Linting certificate" part out, and just emit whatever messages there are. These messages will be seen by regular users, and I think "lint" as a verb isn't widely known outside software development circles. Might be difficult to translate as well.

Reply:
Good idea.